src/pmpo_fortran.f90

```diff
@@ -71,6 +71,22 @@ subroutine polympo_createMPs(mpMesh, numElms, numMPs, mpsPerElm, mp2Elm, isMPAct
     type(c_ptr), intent(in), value :: isMPActive
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief move MPs to a new element, add new MPs, or delete MPs
+  !> @brief the fields associated with the MPs are NOT initialized
+  !> @param mpmesh(in/out) MPMesh object
+  !> @param newNumMPs(in) number of MPs to add to MPMesh
+  !> @param allMP2Elm(in) the target element for each MP
+  !> @param addedMPMask(in) set to 1 each new MP, 0 otherwise
```

Comment:
```suggestion
  !> @param addedMPMask(in) set to 1 for each new MP, 0 otherwise
```
We should document the expected lengths of `allMP2Elm` and `addedMPMask`.

Reply:
This has been updated